src/_macosx.m

```diff
@@ -1064,36 +1060,6 @@ + (WindowServerConnectionManager *)sharedManager
     return sharedWindowServerConnectionManager;
 }
 
-+ (id)allocWithZone:(NSZone *)zone
```

Comment:
https://developer.apple.com/documentation/objectivec/nsobject/1571945-allocwithzone

> Do not override allocWithZone: to include any initialization code. Instead, class-specific versions of init... methods.


OK, good thing we are deleting it?

Reply:
it is also not clear we used this as the actual call site just above uses `[[super allocWithZone:Null] init]` ?

Reply:
This all came in via 7bacdc5c62e661fceee242011799200ea16b8095 with the commit message

> This patch allows interrupts to be delivered once Python is fixed.